### ISSUE TITLE
Deleting a ruleset does not cause a book to become unbaked

### ISSUE THREAD

**Issue by kerwinso** [Connexions/cnx-publishing#179]
Acceptance criteria: "When you delete a ruleset from a book and then tell it to bake, the book becomes unbaked." 

Test steps:
- Ross manually deleted the ruleset for Statistics (30189442-6998-4686-ac05-ed152b91b9de@18.55) in the db.
- Started the bake.
- Bake finished with a state of errored:

```
2017-09-18 14:07:48,546 DEBUG [cnxpublishing][MainThread] Queued for processing module_ident=193771 ident_hash=30189442-6998-4686-ac05-ed152b91b9de@18.55
2017-09-18 14:07:50,360 DEBUG [cnxpublishing][MainThread] Starting baking module_ident=193771 ident_hash=30189442-6998-4686-ac05-ed152b91b9de@18.55
2017-09-18 14:09:36,202 DEBUG [cnxpublishing][MainThread] Exported module_ident=193771 ident_hash=30189442-6998-4686-ac05-ed152b91b9de@18.55
2017-09-18 14:09:36,259 DEBUG [cnxpublishing][MainThread] Finished baking module_ident=193771 ident_hash=30189442-6998-4686-ac05-ed152b91b9de@18.55 with a final state of 'errored'.
```

Webview is still baked: https://qa.cnx.org/contents/MBiUQmmY@18.55:kcV4GRqc@10/Preface

**Comment by kerwinso** [comment]
addressed in #180

**Comment by kerwinso** [comment]
Verified fixed by Ross deleting the ruleset for Stats `30189442-6998-4686-ac05-ed152b91b9de@18.55` and rebaking. [Webview](https://qa.cnx.org/contents/MBiUQmmY@18.55:y0GFmfab@6/Definitions-of-Statistics-Prob) is now properly unbaked.
